data/json/furniture_and_terrain/terrain-walls.json

```
@@ -1323,5 +1323,46 @@
         { "item": "scrap", "count": [ 2, 4 ] }
       ]
     }
+  },
+  {
+    "type": "terrain",
+    "id": "t_drystone_wall",
+    "name": "field stone wall",
+    "description": "A sturdy dry stone wall.  Just rocks fitted together without mortar.",
+    "symbol": "LINE_OXOX",
+    "color": "light_gray",
+    "move_cost": 0,
+    "coverage": 100,
+    "roof": "t_flat_roof",
+    "flags": [ "NOITEM", "WALL", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": {
+      "str_min": 80,
```

```suggestion
      "str_min": 50,
```
80 seems pretty high for a  pile of dry-fitted rocks

---

done